src/Centreon/Application/Controller/MonitoringResourceController.php

```diff
@@ -862,6 +862,31 @@ public function buildServiceUri(int $hostId, int $serviceId, string $tab = self:
         ]);
     }
 
+    /**
+     * Build uri to access meta service panel
+     *
+     * @param integer $metaId
+     * @param string $tab tab name
+     * @return string
+     */
+    public function buildMetaServiceDetailsUri(int $metaId, string $tab = self::TAB_DETAILS_NAME): string
```

Comment:
When is this method used?

Reply:
This method has been added for the Business Activity module as MetaServices can be set as KPI of a BA.